src/main/java/org/sonar/plugins/resharper/ReSharperReportParser.java

```diff
@@ -62,8 +66,11 @@
           if (stream.next() == XMLStreamConstants.START_ELEMENT) {
             String tagName = stream.getLocalName();
 
-            if ("Issue".equals(tagName)) {
-              handleIssueTag();
+            if ("Project".equals(tagName)) {
+              if(handleProjectTag()) {
+                // break after project tag has been handled.
+                break;
```

Comment:
@rikvandenberg several projects can have the same name, so if you break here, you won't import issues from the 2nd, 3rd, and so on occurrences of the same project name